rlberry/manager/tests/test_experiment_manager.py

```diff
@@ -403,3 +405,51 @@ def test_logs(style_log):
     )
     stats.fit()
     evaluate_agents([stats], show=False)
+
+
+def test_save_logger_and_warning(caplog):
+    with tempfile.TemporaryDirectory() as tmpdir:
+        agent_test_name = "test_agent"
+
+        path_to_save = tmpdir + "/" + agent_test_name
+        if os.path.isdir(path_to_save):
+            shutil.rmtree(path_to_save)
```

Comment:
Typically, this can be ignored by the coverage no?

Reply:
yes , but in other PR https://github.com/rlberry-py/rlberry/pull/396